app/components/shared/inputs/TimerInput.vue.ts

```diff
@@ -0,0 +1,132 @@
+import { Component, Prop } from 'vue-property-decorator';
+import { BaseInput } from './BaseInput';
+import { ITimerMetadata } from './index';
+
+let holdTimeout: number = null;
+let holdInterval: number = null;
```

Comment:
I'm a little nervous about these 2 variables with changeable values will be shared between all component's instances. Did you declare it here for any special case?

Reply:
that was copied directly from the website's code